Assets/Scripts/Screeps_API/ScreepsLogin.cs

```diff
@@ -322,7 +320,8 @@ private void LoadServers()
                     }
 
                     var sortedCache = new CacheList();
-                    sortedCache.AddRange(_servers.OrderByDescending(s => s.MMO).ThenBy(s => s.Address.Path).ThenBy(s => s.Address.HostName));
+                    sortedCache.AddRange(_servers.OrderBy(s => s.Type).ThenByDescending(s => s.LikeCount)
+                        .ThenBy(s => s.Address.Path).ThenBy(s => s.Address.HostName));
```

Comment:
I'm not fond of moving the sorting to be based on the order defined inside the ServerType enum.

This should instead be `s.Type == ServerType.Official` and `OrderByDescending`